.github/workflows/ci-backend.yml

```diff
@@ -105,12 +89,12 @@ jobs:
 
             - name: Check migrations
               env:
-                  DATABASE_URL: 'postgres://postgres:postgres@localhost:${{ job.services.postgres.ports[5432] }}/postgres'
+                  DATABASE_URL: 'postgres://posthog:posthog@localhost:5439/posthog'
```

Comment:
I **hate** the 5439 default and have it patched locally. It just adds more jank to our setups/more things to fail when running things locally.

IIRC this was because you had pg installed as a brew package/running as a service. Let's fix it and not do that going forward, wdyt?

Reply:
totally works for me